### ISSUE TITLE
Flag for Connected Investor

### ISSUE THREAD

**Issue by sethmcknight** [SeedCompany/cord-api-v3#2072]

**Comment by CarsonF** [comment]
Duplicate of #2070?

**Comment by sethmcknight** [comment]
@Carson no, this is to display in CORD Field that an Investor is directly connected to the Project/Investor in Salesforce.

**Comment by sethmcknight** [comment]
Dependent on https://github.com/SeedCompany/seed-api/issues/190

**Comment by CarsonF** [comment]
Actually this is independent of that as the data is flowing the other direction. We just need to add a field for this. And then seed API will need a _salesforce webhook_ to listen for changes in Salesforce and forward them to Cord.
